src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.jsx

```diff
@@ -16,12 +18,29 @@ import WebexMeetingInfo from '../WebexMeetingInfo/WebexMeetingInfo';
  * @returns {object} JSX of the component
  */
 export default function WebexInterstitialMeeting({meetingID}) {
+  const {videoPermission} = useMeeting(meetingID);
+  const showModal = videoPermission === 'asking';
+
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-interstitial-meeting`}>
       {meetingID ? (
         <>
           <WebexMeetingInfo className="interstitial-meeting-info" meetingID={meetingID} />
           <WebexLocalMedia className="interstitial-media" meetingID={meetingID} mediaType="video" />
+          {showModal && (
+            <Modal
+              applicationId="app-media-modal"
```

Comment:
do we need this prop? also, `htmlId`? I would prefer to avoid them if possible

Reply:
yes, they seems to be mandatory, if I don't add them the component will display an error in console telling that they are mandatory props